### ISSUE TITLE
XMLHttpRequest deprecation

### ISSUE THREAD

**Issue by Salokyn** [fisharebest/webtrees#2585]
Webtrees version: 1.7.14

Hi,
I get the following message in the Chrome 77 console:

> [Deprecation] Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.

Report:
```json
{
    "columnNumber": 15032,
    "id": "XMLHttpRequestSynchronousInNonWorkerOutsideBeforeUnload",
    "lineNumber": 4,
    "message": "Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.",
    "sourceFile": "https://domain/packages/jquery-2.2.1/jquery.min.js",
    "url": "https://domain/index.php"
}
```

**Comment by fisharebest** [comment]
webtrees 1.7 uses an older version of jQuery, which triggers this error.

It is fixed in webtrees 2.0.

**Comment by fisharebest** [comment]
This issue is fixed in 2.0.0, which is now released.
